tests/ImageSharp.Tests/Formats/Jpg/JpegEncoderTests.cs

```diff
@@ -181,14 +182,37 @@ public async Task Encode_IsCancellable(JpegEncodingColor colorType)
             }
         });
 
-        using var image = new Image<Rgba32>(5000, 5000);
+        using Image<Rgba32> image = new(5000, 5000);
         await Assert.ThrowsAsync<TaskCanceledException>(async () =>
         {
-            var encoder = new JpegEncoder() { ColorType = colorType };
+            JpegEncoder encoder = new() { ColorType = colorType };
             await image.SaveAsync(pausedStream, encoder, cts.Token);
         });
     }
 
+    // https://github.com/SixLabors/ImageSharp/issues/2595
+    [Theory]
+    [WithFile(TestImages.Jpeg.Baseline.ForestBridgeDifferentComponentsQuality, PixelTypes.Bgra32)]
```

Comment:
I'm explicitly using a real file here rather than a generated one to make it easier to do visual verification.